flag/service/service.go

```diff
@@ -1,7 +1,7 @@
 package service
 
 import (
-	"github.com/giantswarm/operatorkit/v8/pkg/flag/service/kubernetes"
+	"github.com/giantswarm/operatorkit/v7/pkg/flag/service/kubernetes"
```

Comment:
Why the rollback?
Looks like the v8 never even existed, latest release is 7.1.0 :astonished: (https://github.com/giantswarm/operatorkit/blob/master/CHANGELOG.md) - how did it work with v8? :scream_cat:

Reply:
Well it did not work with 8.0.0 that's the thing. We had a v8 release but it was reverted. I bumped it with renovate last week though

Reply:
I'm not sure why the renovate pr built then